contributing.md

```diff
@@ -69,10 +69,8 @@ to boot a rails app under the control of the master process.
 
 ### Building
 
-Just run `make`, basically. The `Makefile` should be an easy read. `make
-darwin` and related tasks will just build the go binaries to `./build`, while
-`make gem` will build the rubygem from the files currently in `./build`. `make
-all` does the whole shebang, but takes a few seconds to complete.
+Just run `make`, which would build the go binaries to `./build`
```

Comment:
This no longer outputs anything to `build`, it goes straight to `rubygem/build`.